keras/engine/base_layer.py

```diff
@@ -3145,6 +3147,18 @@ def __setattr__(self, name, value):
             name, value
         )
 
+    def _update_trackables(self):
+        """This method loops over tracked objects to update tracked variables
+        for wrapped lists/dicts that may have been updated after initialization
+        (Issue #16797)
+        """
```

Comment:
Just "Track variables added to lists/dicts after creation." no need to reference the issue

Reply:
Ok! I've done both things. I've checked that the unit test works and that, if tested on, say, Google Colab with current Keras version, it fails.